databricks/koalas/frame.py

```diff
@@ -484,10 +484,16 @@ def calculate_columns_axis(*cols):
         else:
             raise ValueError("No axis named %s for object type %s." % (axis, type(axis)))
 
+    def _kser_for(self, label):
```

Comment:
@ueshin, out of curiosity, which code path was the concern this shortcut avoids specifically? and ..  are we just able to move this shortcut into `DataFrame.__getitem__` or `LocIndexer`?

Reply:
`LocIndexer._get_from_multiindex_column` could be an overhead, especially for deep level labeled columns.
As for the movement, we need to look up the column if the input is from users, so I don't think we should move this into `DataFrame.__getitem__` or `LocIndexer`.

Reply:
Hm .. maybe I missed something but the number of columns are expected to be small. Wouldn't it the performance improvement is trivial? I was wondering if it's worthwhile having a shortcut which makes a bit difficult to read. I myself would be confused when to use `self._kser_for(label)` vs `self[label]`.

Reply:
Probably the PR title is confusing you?
This PR is just moving the way to create Series with a proper label into DataFrame.
Basically we can just use `self._kser_for(label)`, and if the label is from user input, we should use `self[label]`.